tui/show.go

```diff
@@ -280,3 +271,24 @@ func (ui *UI) showHelp() {
 	ui.pages.AddPage("help", flex, true, true)
 	ui.app.SetFocus(text)
 }
+
+func formatHelpTextFor(ui *UI) string {
```

Comment:
```suggestion
func (ui *UI) formatHelpTextFor string {
```

Adding this as a method is IMO more idiomatic.